api/pom.xml

```diff
@@ -96,6 +96,11 @@
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
I don't like the idea of pulling more Hadoop dependencies into the API. Can we do without it?